### PR TITLE
[eks-argo] Add instructions for how to publish argo events from outside cluster

### PR DESCRIPTION
**Pull request by jackie-ob**

### CODE REVIEW

File: examples/eks_argo/eks.tf [hunk 1]
```diff
@@ -4,7 +4,7 @@ module "eks" {
   version = "17.23.0"
 
   cluster_name    = local.cluster_name
-  cluster_version = "1.21"
+  cluster_version = "1.24"
```
Comment (jackie-ob) [inline examples/eks_argo/eks.tf:7]: Drive-by:  this is a must do.... 1.21 no longer accepted by AWS.

**Review by savingoyal — APPROVED** [review]
